src/markProcessor.ts

```diff
@@ -82,6 +82,10 @@ export class MarkProcessor {
     this.index += 1
   }
 
+  unshift(): void {
+    this.index -= 1
+  }
+
```

Comment:
Not used anywhere?

Reply:
This is used in `func_call` in the `SELECTOR_BUILDER`. I'll see if there's a way to avoid needing to use it.